extensions/bible/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "bible",
+  "title": "Bible",
+  "description": "An extension for all things bible-related.",
```

Comment:
Can you rephrase this line?

Reply:
Description updated. I just described the main function of the extension (search) for now. I will update it when I add more features